src/services/ledger/index.ts

```diff
@@ -378,15 +368,15 @@ export const LedgerService = (): ILedgerService => {
 export const translateToLedgerTx = (
   tx: ILedgerTransaction,
 ): LedgerTransaction<WalletCurrency> => ({
-  id: tx.id as LedgerTransactionId,
+  id: fromObjectId<LedgerTransactionId>(tx._id + ""),
```

Comment:
why is the `+ ""` necessary? we shouldn't need string casting?

Reply:
_id can be undefined (typescript error), in theory we should not have transactions without id here (this is medici internal).

alternative 
```
fromObjectId<LedgerTransactionId>(tx._id || "")
```

Reply:
I prefer the alternative. 

also maybe we should have a comment on why we have this `|| ""`

Reply:
I did the update for the alternative, maybe is clear that _id interface define it as undefined too  https://github.com/GaloyMoney/galoy/pull/1406/commits/e77103c77d9694e935f3d0246a4bf12a22ce4ea5